spring-data-relational/src/main/java/org/springframework/data/relational/core/mapping/AggregatePath.java

```diff
@@ -227,6 +245,9 @@ default Stream<AggregatePath> stream() {
 	 */
 	AggregatePath getIdDefiningParentPath();
 
+	@Nullable
+	AggregatePath getTail();
```

Comment:
Isn't an aggregate path always at least one element? If so, why is `getTail` nullable? Also, a bit of documentation (including `@since`) helps in avoiding questions.

Reply:
No, AggregatePath might have zero elements, in which case it is referring to the aggregate root.

Reply:
Done